scene/gui/gradient_edit.cpp

```diff
@@ -367,6 +367,13 @@ void GradientEdit::_notification(int p_what) {
 			draw_line(Vector2(-1, -1), Vector2(-1, h + 1), Color(1, 1, 1, 0.6));
 		}
 	}
+
+	if (p_what == NOTIFICATION_VISIBILITY_CHANGED) {
+
+			if (!is_visible()) {
```

Comment:
You're using two tabs as indentation level, while the rest of the code uses one, so `clang-format` reports a style error.